srcpkgs/byte/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'byte'
+pkgname=byte
+version=0.3.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config vala glib-devel"
+makedepends="libgee08-devel granite-devel sqlite-devel libsoup-devel
+ json-glib-devel gst-plugins-base1-devel taglib-devel"
+short_desc="Music Player designed for pantheon"
+maintainer="linarcx <linarcx@riseup.net>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/alainm23/byte"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=a55449f7869b999517495154bdb31474e9b04c1e30ba8f8b8b0f6eef6306527c
+
+post_install() {
+	vinstall ${FILESDIR}/com.github.alainm23.byte.desktop 644 usr/share/applications
```

Comment:
don't bundle this file if it's contained in [the sources](https://github.com/alainm23/byte/blob/d862d5f73d870572440b389d9ed94d1594d9530d/data/com.github.alainm23.byte.desktop.in)